# Disable the dependency on numpy in setup.py. The Python installation is assumed to already have
# a numpy module, but for some reason fdstools is trying to reinstall it anyway.
--- fdstools-master-3a4956538be6d0b899d784f4967ddc6ff57c7d82/setup.py	2016-03-22 17:57:38.000000000 +0100
+++ fdstools-master-3a4956538be6d0b899d784f4967ddc6ff57c7d82.new/setup.py	2016-06-02 11:26:32.000000000 +0200
@@ -1,7 +1,8 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages
 
-requires = ["numpy"]
+#requires = ["numpy"]
+requires = []
 
 # Python 2.6 does not include the argparse module.
 try:
@@ -49,4 +50,4 @@
             "fdstools=fdstools.fdstools:main"
         ]
     }
-)
\ No newline at end of file
+)
